bundle/config/mutator/apply_presets.go

```diff
@@ -222,6 +222,19 @@ func (m *applyPresets) Apply(ctx context.Context, b *bundle.Bundle) diag.Diagnos
 		dashboard.DisplayName = prefix + dashboard.DisplayName
 	}
 
+	// Apps: Prefix
+	for key, app := range r.Apps {
+		if app == nil || app.App == nil {
+			diags = diags.Extend(diag.Errorf("app %s is not defined", key))
+			continue
+		}
+
+		app.Name = textutil.NormalizeString(prefix + app.Name)
+		// Normalize the app name to ensure it is a valid identifier.
+		// App supports only alphanumeric characters and hyphens.
+		app.Name = strings.ReplaceAll(app.Name, "_", "-")
+	}
+
```

Comment:
We shouldn't auto-prefix apps.

Similar to UC, apps are addressable only by their name. It's not only for display. Updating it implies a recreate. Isolation of app instances across multiple development deployments is still necessary, but we could emit a recommendation to include the current user name or variable in the app name if there isn't one.

On a related note, the name is limited to max 30 characters, which you can run into quickly with a long user name and our default name prefix.